### PR TITLE
feat: basic execute watcher code

### LINKED ISSUE #1: setup notify-watcher core
- libs for use by watchers
- error codes so watcher can report errors

### PR DESCRIPTION
**Pull request by negebauer**
Closes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,19 @@
-# execute-watcher
\ No newline at end of file
+# Notify watcher core
+
+Utilities for running Notify watcher watcher's
```
Comment (sasalatart) [inline README.md]:
```suggestion
Utilities for running Notify watcher's watchers
```

File: README.md [hunk 2]
```diff
@@ -1 +1,19 @@
...
+
+Run a watcher by using `execute`
+
+```js
+const { execute } = require('@notify-watcher/core');
+
+// Replace with your watcher
+const watcher = {}
+
+// Provided required data to watcher
```
Comment (sasalatart) [inline README.md]: Maybe this line can be explained a bit more

Comment (sasalatart) [reply README.md]: And these same options are then passed on to the same watchers you are giving here. I don't understand this part

Comment (negebauer) [reply README.md]: The server uses `execute` to call the watchers

**Pull request merged**